### PR TITLE
include avro in the druid tarball

### PR DESCRIPTION
**Pull request by fjy**

### CODE REVIEW

File: distribution/pom.xml [hunk 1]
```diff
@@ -83,13 +87,13 @@
...
                                 <argument>io.druid.extensions:druid-kafka-indexing-service</argument>
                                 <argument>-c</argument>
-                                <argument>io.druid.extensions:mysql-metadata-storage</argument>
-                                <argument>-c</argument>
                                 <argument>io.druid.extensions:druid-lookups-cached-global</argument>
                                 <argument>-c</argument>
-                                <argument>io.druid.extensions:postgresql-metadata-storage</argument>
                                 <argument>-c</argument>
                                 <argument>io.druid.extensions:druid-s3-extensions</argument>
+                                <argument>io.druid.extensions:mysql-metadata-storage</argument>
```
Comment (gianm) [inline distribution/pom.xml]: these look out of order

Comment (fjy) [inline distribution/pom.xml]: oops fixed

**Comment by drcrallen** [comment]
👍

**Comment by gianm** [comment]
@fjy not sure why travis "push" check keeps failing, could you try squashing your commits, maybe that'll help

**Comment by gianm** [comment]
👍 other than that though

**Comment by nishantmonu51** [comment]
👍

**Pull request merged**